.ci/run-e2e-tests.sh

```diff
@@ -13,7 +13,8 @@ if [ "${TEST_GROUP}" = "es" ]; then
     make e2e-tests-es
 elif [ "${TEST_GROUP}" = "es-otel" ]; then
     echo "Running elasticsearch tests with OTEL collector"
-    export USE_OTEL_COLLECTOR=true
+    export SPECIFY_OTEL_IMAGES=true
```

Comment:
Is it about using the OpenTelemetry images instead of Jaeger v1?

Reply:
Yes